### PR TITLE
Fixed server error occuring when timestamp is invalid while sending mentorship relation request

### LINKED ISSUE #383: Internal Server Error when sending relationship request
**Describe the bug**
I encountered a strange and possibly unintended behavior while developing Mentorship Client. I accidentaly multiplied timestamp by 1000 (I actually wanted to divide it by 1000).

**To Reproduce**
1. Go to http://systers-mentorship-dev.eu-central-1.elasticbeanstalk.com/
2. Open Mentorship Relation tab
3. Expand first endpoint – `POST /mentorship_relation/send_request`
4. Enter valid mentor_id and mentee_id, pass valid JWT and valid notes.
5. Pass too big value as end_date.
6. Admire undocumented status code 500.

**Expected behavior**
`400 Bad Request` should be returned with helpful message e.g "end_date too big".

**Screenshots**
internal server error

### PR DESCRIPTION
**Pull request by bartekpacia**
### Description
Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes #383

### Type of Change:
- Code
- Quality Assurance

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
Tested manually with emulator device + added tests that proves that the fix works. Also tested manually with Swagger.
fixed 500

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: tests/mentorship_relation/test_dao_creation.py [hunk 1]
```diff
@@ -219,6 +219,25 @@ def test_dao_create_mentorship_relation_with_mentor_already_in_relation(self):
...
+            end_date=1580338800000000,
+            notes=self.notes_example,
+            tasks_list=TasksListModel()
+        )
+
+        # Use DAO to create a mentorship relation
+
+        result = dao.create_mentorship_relation(self.first_user.id, data)
+
+        print(result)
```
Comment (isabelcosta) [inline tests/mentorship_relation/test_dao_creation.py]: Can you remove this print, it was used for testing, but it does not need to be here right?

**Review by isabelcosta — CHANGES_REQUESTED** [review]
This is a great contribution :) Thank you for it @bartekpacia 
Just left a little comment, will be happy to approve on you remove a `print()` statement

File: tests/mentorship_relation/test_dao_creation.py (hunk 1 above)
Comment (bartekpacia) [reply tests/mentorship_relation/test_dao_creation.py]: Sure, forgot to remove!

Comment (bartekpacia) [reply tests/mentorship_relation/test_dao_creation.py]: Removed :D

**Review by isabelcosta — APPROVED** [review]
Good work @bartekpacia !
Thank you for your contribution :)

**Pull request merged**